includes/EDD_SL_Plugin_Updater.php

```diff
@@ -307,6 +314,8 @@ private function api_request( $_action, $_data ) {
             $request = false;
         }
 
+	set_transient( $cache_key, $request, 60*60*24 );
```

Comment:
Can we use `DAY_IN_SECONDS` instead of `60*60*24`? It's just easier to read on a quick pass through.